src/thunderbots/software/ai/hl/stp/evaluation/player.h

```diff
@@ -0,0 +1,40 @@
+#ifndef AI_HL_STP_EVALUATION_PLAYER_H_
+#define AI_HL_STP_EVALUATION_PLAYER_H_
+
+#include "geom/point.h"
+
+/**
+ * Contain independent Evaluation function to evaluate whether player orientation at
+ * position is within threshold
+ */
+
+namespace AI
+{
+    namespace HL
+    {
+        namespace STP
+        {
+            namespace Evaluation
```

Comment:
We're no longer using the `AI::HL::STP::Evaluation` namespace in our new code. This should just be in namespace `Evaluation`.